CMakeLists.txt

```diff
@@ -1724,8 +1724,8 @@ if(ENGINEPRIME)
     set(DJINTEROP_LIBRARY "lib/${CMAKE_STATIC_LIBRARY_PREFIX}djinterop${CMAKE_STATIC_LIBRARY_SUFFIX}")
     file(MAKE_DIRECTORY "${CMAKE_CURRENT_BINARY_DIR}/download")
     ExternalProject_Add(libdjinterop
-      URL "https://github.com/xsco/libdjinterop/archive/0.14.6.tar.gz"
-      URL_HASH SHA256=db2f57f6c06c801d1785280ede0f032d7280bedd72f2a30bc263a272e3269587
+      URL "https://github.com/xsco/libdjinterop/archive/0.15.1.tar.gz"
```

Comment:
Out of scope: Did consider switching to GIT_REPOSITORY + GIT_TAG? Just used this variant recently in a project and it seems to be more concise.

Reply:
This did originally use `GIT_REPOSITORY` etc. but I changed it in a (sadly misguided) attempt to get the Ubuntu build working.  I agree with you that `GIT_REPOSITORY` is more clear vs. a download link on GitHub.  It's also easier during development to temporarily point to a branch (e.g. when I'm testing out a new version of libdjinterop that hasn't been merged yet).

I'll make a note to change this back at some point!

Reply:
#3779 

I wasn't aware of the Debian/Ubuntu build dependencies. The Fedora build system requires you to manually upload and hash sources in advance.

Reply:
There is a more full discussion in https://github.com/mixxxdj/mixxx/pull/3620

In short, there was no need to change from `GIT_REPOSITORY` to plain old http downloads.  The Ubuntu build should not be downloading sources of dependencies from GitHub, and should instead be using packages.  To that end, libdjinterop [now has its own PPA](https://launchpad.net/~xsco/+archive/ubuntu/djinterop), and I either Mixxx's PPA can use the same packages, or make a dependency on the libdjinterop PPA.

This reminds me to talk to Daniel about agreeing the approach, which I'll do on Zulip now.